content/events/current.md

```diff
@@ -8,12 +8,45 @@ Coming soon to a virtual venue near you. Date: TBD, some weekend in September.
 
 # Past events
 
-#### 8/2/20 Annual Meeting
+#### 2020-08-02 Annual Meeting
```

Comment:
I took the liberty of converting the dates to ISO format. :) I remember getting so confused when I lived in Switzerland about which was month and which was day that I somehow got allergic. Happy to revert if you think it was better before.

Reply:
no strong feelings either way